ruby-challenges.rb

```diff
@@ -11,20 +11,84 @@
 # Expected output: ['tea', 'water', 'soda water']
 
 
+def find_words(words, letter)           
+filtered_words = []                     
+
+words.each do |word|                   
+    if word.include?(letter)          
+        filtered_words << word         
```

Comment:
Same thing here as in the Javascript section, how can you remove the need for the filtered_words variable? Hint, take a look at `.select`.